### PR TITLE
LVM2: Update to 2.02.80

### PR DESCRIPTION
**Pull request by neheb**
Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer: @dangowrt 
Compile tested: ipq806x

### CODE REVIEW

File: utils/lvm2/Makefile [hunk 1]
```diff
@@ -9,14 +9,14 @@
...
+PKG_VERSION:=2.02.180
+PKG_RELEASE:=1
 PKG_LICENSE:=GPL-2.0 LGPL-2.1
 
 PKG_SOURCE:=$(PKG_NAME).$(PKG_VERSION).tgz
-PKG_SOURCE_URL:=ftp://sources.redhat.com/pub/lvm2/releases \
-		http://ftp.gwdg.de/pub/linux/sources.redhat.com/lvm2/
-PKG_HASH:=4025a23ec9b15c2cb7486d151c29dc953b75efc4d452cfe9dbbc7c0fac8e80f2
+PKG_SOURCE_URL:=https://ftp.gwdg.de/pub/linux/sources.redhat.com/lvm2 \
+		ftp://sources.redhat.com/pub/lvm2/releases
```
Comment (pprindeville) [inline utils/lvm2/Makefile]: Please put `sources.redhat.com` first as they have several mirrors.

File: utils/lvm2/Makefile [hunk 2]
```diff
@@ -40,8 +40,8 @@ define Package/lvm2
   CATEGORY:=Utilities
   SUBMENU:=Disc
   TITLE:=The Linux Logical Volume Manager
-  URL:=http://sourceware.org/lvm2/
-  DEPENDS:=+libdevmapper +libblkid +libreadline +libncurses
+  URL:=https://sourceware.org/lvm2/
+  DEPENDS:=+libdevmapper +libblkid +libreadline +libncurses +libaio
```
Comment (pprindeville) [inline utils/lvm2/Makefile]: Why the new dependency?

And wouldn't this require new `CONFIGURE_ARGS`?

Comment (neheb) [reply utils/lvm2/Makefile]: Build was failing on it.

There's a new version of LVM2. I will look into this soon.

Comment (pprindeville) [reply utils/lvm2/Makefile]: Asked and answered:

https://sourceware.org/git/?p=lvm2.git;a=blob;f=WHATS_NEW;h=0a0b6e925c0a967be4514f13abfca42f3d4b019e;hb=HEAD#l24

File: utils/lvm2/Makefile (hunk 1 above)
Comment (neheb) [reply utils/lvm2/Makefile]: Ah looks like it's reachable over HTTP but not HTTPS. I'll remove the ftp.

File: utils/lvm2/Makefile (hunk 2 above)
Comment (neheb) [reply utils/lvm2/Makefile]: Ah yes. I recall looking into the configure script to look for a way to get rid of it.

**Comment by neheb** [comment]
Switched to sourceware URL for consistency.

**Comment by dangowrt** [comment]
Thanks!

**Pull request merged**